src/window.rs

```diff
@@ -868,6 +881,24 @@ pub trait Window: AsAny + Send + Sync {
     /// - **Wayland / iOS / Android / Web:** Unsupported.
     fn is_visible(&self) -> Option<bool>;
 
+    /// Sets whether the window can be focused or not.
+    ///
+    /// The default is `true`.
+    ///
+    /// - **Windows:** Supported.
+    #[inline]
+    pub fn set_focusable(&self, focusable: bool) {
```

Comment:
I feel like the _semantics_ of the feature aren't clear just from reading the description here. Perhaps you could expand on how it works? Or state the example you gave in the PR (useful for ...).

In the absence of clear cross-platform semantics, I'd like if this could link to the underlying implementations (e.g. link to [`shouldDelayWindowOrderingForEvent:`](https://developer.apple.com/documentation/appkit/nsview/shoulddelaywindowordering(for:)?language=objc) for macOS, and others for the other platforms).